src/count/demos/src/auto_refresh/demo.mustache

```diff
@@ -0,0 +1,3 @@
+<div data-o-component="o-comment-count" data-o-comment-count-config-article-id="{{articleId}}" data-o-comment-count-config-auto-refresh="true">
```

Comment:
I think we will want to move the demos to the root level so they are considered part of o-comments